shared/common-adapters/box.desktop.js

```diff
@@ -27,9 +27,27 @@ const injectGaps = (Component, _children, gap, gapStart, gapEnd) => {
 
 class Box2 extends React.Component<Box2Props> {
   render() {
-    const horizontal = this.props.direction === 'horizontal'
+    let directionStyle
+    let horizontal = this.props.direction === 'horizontal' || this.props.direction === 'horizontalReverse'
+    switch (this.props.direction) {
+      case 'horizontal':
+        directionStyle = globalStyles.flexBoxRow
+        break
+      case 'horizontalReverse':
+        directionStyle = {display: 'flex', flexDirection: 'row-reverse'}
```

Comment:
this has to pull from styles, not make a new object every time or you'll re render over and over